x/incentives/keeper/gauge_test.go

```diff
@@ -8,22 +8,60 @@ import (
 	lockuptypes "github.com/osmosis-labs/osmosis/x/lockup/types"
 )
 
+// TestDistribute tests that when the distribute command is executed on
+// a provided gauge,
+func (suite *KeeperTestSuite) TestDistribute() {
+	twoLockupUser := userLocks{
+		lockDurations: []time.Duration{time.Second, 2 * time.Second},
+		lockAmounts:   []sdk.Coins{defaultLPTokens, defaultLPTokens},
+	}
+	defaultGauge := perpGaugeDesc{
+		lockDenom:    defaultLPDenom,
+		lockDuration: defaultLockDuration,
+		rewardAmount: sdk.Coins{sdk.NewInt64Coin(defaultRewardDenom, 3000)},
+	}
+	oneKRewardCoins := sdk.Coins{sdk.NewInt64Coin(defaultRewardDenom, 1000)}
+	twoKRewardCoins := oneKRewardCoins.Add(oneKRewardCoins...)
+	tests := []struct {
+		users           []userLocks
+		gauges          []perpGaugeDesc
+		expectedRewards []sdk.Coins
+	}{
+		{
+			users:           []userLocks{oneLockupUser, twoLockupUser},
+			gauges:          []perpGaugeDesc{defaultGauge},
+			expectedRewards: []sdk.Coins{oneKRewardCoins, twoKRewardCoins},
+		},
+	}
+	for _, tc := range tests {
+		suite.SetupTest()
+		gauges := suite.SetupGauges(tc.gauges)
```

Comment:
I was debugging this part and noticed that interestingly, three locks are existing despite we only created two locks above. Two were as expected, and the one extra one had the same defaultLPDenom and had the duration of `net/http.http2firstSettingsTimeout (2000000000)` has this lock been always existing?

Reply:
Wait we created three locks tho, the one lockup user created one lock, and the two lockup user creates two locks